zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -480,6 +502,15 @@ impl<F: Field> BytecodeCircuitConfig<F> {
             .evm_word()
             .map(|challenge| rlc::value(EMPTY_CODE_HASH_LE.as_ref(), challenge));
```

Comment:
The old `empty_hash` is rlc based. We can safely remove it.

Reply:
Done